src/api/VoteMonitor.Entities/NoteAttachment.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace VoteMonitor.Entities
+{
+    public class NoteAttachment
+    {
+        public NoteAttachment(string notePath)
+            => NotePath = notePath;
+
+        [Key, Required]
+        public int Id { get; set; }
+
+        [Required, MaxLength(1000)]
+        public string NotePath { get; set; }
+
+        [JsonIgnore]
```

Comment:
I think we won't be needing these annotations if we return DTOs (in this case, we should use `NoteModel` as a return type for `Upload`)

Reply:
Some of those Annotations are required by EF Core.
Removed JsonIgnore tho since we're using NoteModel.